deployments/compose/hive/wait-for

```diff
@@ -0,0 +1,79 @@
+#!/bin/sh
```

Comment:
We have a `wait-for` script in this repo, can we reuse it?

Reply:
I'll find an alternative and remove this copy of wait-for script